src/command/CommandList.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  List,
+  Datagrid,
+  TextField,
+  ReferenceField,
+  NumberField,
+  DateField,
+  ArrayField,
+  SingleFieldList,
+  ChipField,
+} from "react-admin";
+
+export const CommandList = () => (
+  <List resource="commands" title="Orders">
+    <Datagrid rowClick="edit">
+      <TextField source="id" />
+      <TextField source="reference" />
+      <DateField source="date" />
+      <ReferenceField source="customer_id" reference="customers">
+        <TextField source="id" />
+      </ReferenceField>
+      <ArrayField source="basket">
+        <SingleFieldList>
+          <ChipField source="product_id" />
+        </SingleFieldList>
+      </ArrayField>
```

Comment:
You should avoid this kind of fields in list. There might be many products in the basket

Reply:
Fixed in https://github.com/slax57/admin-e-commerce-demo/pull/2 🙂